tests/components/vevor_heater/test_config_flow.py

```diff
@@ -0,0 +1,113 @@
+"""Unit tests for Vevor BLE Heater config flow."""
+from unittest.mock import patch
+
+from home_assistant_bluetooth import BluetoothServiceInfo
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.vevor_heater.config_flow import CannotConnect
+from homeassistant.components.vevor_heater.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
+
+
+@pytest.fixture(autouse=True)
+def mock_bluetooth(enable_bluetooth):
+    """Mock bluetooth for all tests in this module."""
+
+
+async def test_form_cannot_connect(hass: HomeAssistant) -> None:
+    """Test we successfully create the device if we can't connect to it."""
+
+    with patch(
+        "homeassistant.components.vevor_heater.config_flow.validate_device",
```

Comment:
Please patch the pypi code and avoid patching the integration (except for async_setup_entry)

Reply:
Done.